Jenkinsfile

```diff
@@ -28,10 +28,15 @@ pipeline {
 
         stage('Build') {
             steps {
-                sh "make build-image build-minimal-assisted-iso-generator-image"
+                sh "skipper make build-image build-minimal-assisted-iso-generator-image"
                 sh "make jenkins-deploy-for-subsystem"
                 sh "kubectl get pods -A"
             }
+            post {
```

Comment:
what will happen if the build fails?

Reply:
It tries also (because it's `always`) and it would find out there are no files.